.travis.yml

```diff
@@ -13,4 +13,5 @@ install:
   - make init
 script:
   - make build test lint coverage
+  - make docker-all
```

Comment:
I don't think so, you asked: Should this be replacing, or just being added? (followed by "if yes", ...)

the way I read it is: text in its literal form doesn't state preference but the fact of raising this, kind of does :)

Anyway, with the literal interpretation I think @krancour didn't express preference one way or another, merely making observation and observing possible consequence of it.

I am stating preference: add for now, not replace.